File: JAVA/Aksesoris.java
public class Aksesoris extends Petshop {

    private String jenis, bahan, warna;

    public Aksesoris(){
    }

    public String getJenis() {
        return jenis;
    }


    public void setJenis(String jenis) {
        this.jenis = jenis;
    }


    public String getBahan() {
        return bahan;
    }


    public void setBahan(String bahan) {
        this.bahan = bahan;
    }


    public String getWarna() {
        return warna;
    }


    public void setWarna(String warna) {
        this.warna = warna;
    }

    public int get_ljenis(){
        int panjang = this.jenis.length();
        return panjang;
    }
    public int get_lbahan(){
        int panjang = this.bahan.length();
        return panjang;
    }
    public int get_lwarna(){
        int panjang = this.warna.length();
        return panjang;
    }
}

File: CPP/Aksesoris.cpp
#include <iostream>
#include <string>
#include "Petshop.cpp"

using namespace std;

class Aksesoris: public Petshop
{
private:
    string jenis, bahan, warna;
public:
    Aksesoris(){
        this->jenis = "";
        this->bahan = "";
        this->warna= "";
    };
    Aksesoris(string jenis, string bahan, string warna){
        this->jenis = jenis;
        this->bahan = bahan;
        this->warna = warna;
    }

    void set_jenis(string jenis){
        this->jenis = jenis;
    }
    void set_bahan(string bahan){
        this->bahan = bahan;
    }
    void set_warna(string warna){
        this->warna = warna;
    }

    string get_jenis(){
        return this->jenis;
    }
    string get_bahan(){
        return this->bahan;
    }
    string get_warna(){
        return this->warna;
    }

    int get_ljenis(){
        int panjang = this->jenis.length();
        return panjang;
    }
    int get_lbahan(){
        int panjang = this->bahan.length();
        return panjang;
    }
    int get_lwarna(){
        int panjang = this->warna.length();
        return panjang;
    }


    ~Aksesoris(){
    };
};
